src/arch/Threads/Threads_Pthreads.cpp

```diff
@@ -208,10 +208,11 @@ MutexImpl *MakeMutex( RageMutex *pParent )
 #if defined(UNIX)
 #include <dlfcn.h>
 #include "arch/ArchHooks/ArchHooks_Unix.h"
-#elif defined(MACOSX)
-typedef int clockid_t;
-static const clockid_t CLOCK_REALTIME = 0;
-static const clockid_t CLOCK_MONOTONIC = 1;
+// commented out to allow comilation on macOS 10.12.  -dguzek
```

Comment:
This is going to sound strange, but...that specific file did not show up for me in Xcode when I cleaned my cmake cache. Maybe something had changed to cause that file to now be included for Mac OS X 10.12?

As such, I can only verify that it works because it's not included for me.